content/id/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,414 @@
+### Install packages to allow apt to use a repository over HTTPS
+apt-get update && apt-get install -y \
+  apt-transport-https ca-certificates curl software-properties-common gnupg2
+```
+
+```shell
+# Add Docker’s official GPG key:
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+```
+
+```shell
+# Add the Docker apt repository:
+add-apt-repository \
+  "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) \
+  stable"
+```
+
+```shell
+# Install Docker CE
+apt-get update && apt-get install -y \
+  containerd.io=1.2.13-2 \
+  docker-ce=5:19.03.11~3-0~ubuntu-$(lsb_release -cs) \
+  docker-ce-cli=5:19.03.11~3-0~ubuntu-$(lsb_release -cs)
+```
+
+```shell
+# Set up the Docker daemon
+cat > /etc/docker/daemon.json <<EOF
+{
+  "exec-opts": ["native.cgroupdriver=systemd"],
+  "log-driver": "json-file",
+  "log-opts": {
+    "max-size": "100m"
+  },
+  "storage-driver": "overlay2"
+}
+EOF
+```
+
+```shell
+mkdir -p /etc/systemd/system/docker.service.d
+```
+
+```shell
+# Restart Docker
+systemctl daemon-reload
+systemctl restart docker
+```
+{{% /tab %}}
+{{% tab name="CentOS/RHEL 7.4+" %}}
+
+```shell
+# (Install Docker CE)
+## Set up the repository
+### Install required packages
+yum install -y yum-utils device-mapper-persistent-data lvm2
+```
+
+```shell
+## Add the Docker repository
+yum-config-manager --add-repo \
+  https://download.docker.com/linux/centos/docker-ce.repo
+```
+
+```shell
+# Install Docker CE
+yum update -y && yum install -y \
+  containerd.io-1.2.13 \
+  docker-ce-19.03.11 \
+  docker-ce-cli-19.03.11
+```
+
+```shell
+## Create /etc/docker
+mkdir /etc/docker
+```
+
+```shell
+# Set up the Docker daemon
+cat > /etc/docker/daemon.json <<EOF
+{
+  "exec-opts": ["native.cgroupdriver=systemd"],
+  "log-driver": "json-file",
+  "log-opts": {
+    "max-size": "100m"
+  },
+  "storage-driver": "overlay2",
+  "storage-opts": [
+    "overlay2.override_kernel_check=true"
+  ]
+}
+EOF
+```
+
+```shell
+mkdir -p /etc/systemd/system/docker.service.d
+```
+
+```shell
+# Restart Docker
+systemctl daemon-reload
+systemctl restart docker
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+Jika kamu layanan Docker berjalan pada saat memulai pertama (_boot_), jalankan perintah ini:
+
+```shell
+sudo systemctl enable docker
+```
+
+Silahkan merujuk pada [Panduan resmi instalasi Docker](https://docs.docker.com/engine/installation/)
+untuk informasi lebih lanjut.
+
+## CRI-O
+
+Bagian ini mencakup langkah-langkah yang diperlukan untuk menginstal `CRI-O` sebagai _runtime_ CRI.
+
+Gunakan perintah-perinath berikut untuk menginstal CRI-O pada sistem kamu:
+
+{{< note >}}
+Versi mayor dan minor dari CRI-O harus sesuai dengan versi mayor dan minor dari Kubernetes.
+Untuk informasi lebih lanjut, lihatlah [Matriks kompatibilitas CRI-O](https://github.com/cri-o/cri-o).
+{{< /note >}}
+
+### Prasyarat
+
+```shell
+modprobe overlay
+modprobe br_netfilter
+
+# Set up required sysctl params, these persist across reboots.
```

Comment:
you can translate the comment as well

Reply:
Done in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1